packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1523,16 +1524,19 @@ export class AudiusAPIClient {
     currentUserId,
     userId,
     offset = 0,
-    limit = 100
+    limit = 100,
+    sortMethod = ''
```

Comment:
no need for default arg, since '' will || undefined in the params arg

Reply:
so can just be sort_method: sortMethod, and with that i think you can remove the bottom line too so in the end im thinking: 
```
  let params = {
      user_id: encodedCurrentUserId || undefined,
      limit,
      offset,
      sort_method: sortMethod
    }
```